SharedSource/Core/FunctionalProgramming/EnumerableExtensions.cs

```diff
@@ -249,7 +249,7 @@ public static IEnumerable<TSource> CreateSequence<TSource> (this TSource source,
         this TSource source,
         Func<TSource, TSource> nextElementSelector,
         Func<TSource, bool> predicate,
-        [CanBeNull] IEqualityComparer<TSource> equalityComparer,
+        [CanBeNull] IEqualityComparer<TSource>? equalityComparer,

```

Comment:
We decided to leave them in, but I can't remember the reasoning - I guess it was the need of nullability info for the projects, due to a resharper issue/missing feature. If the need of adapting **ImplicitNullability** arises, wouldn't it be beneficial to add the attributes as a post-compilation step, matching the generated roslyn nullability attributes?

Reply:
@iam-mholle 
not all projects will use C# 8 (or a c# 8-compiler-based Version of Visual Studio)
Wasn't there also the problem that R# itself ignored NRTs in base-projects if the own project isn't NRT enabled?

So, we need R# annotations. How to get them? Option one, leave them in. Problem is, for the 'big' code base, that's going to grow old, fast because we'd also have to maintain them forward. Option two, we update the ImplictNullablity addin. I do not want to do a post-compile (also that's an external decision, too). So, the R# addin would be a much simpler solution.